templates/mod.html

```diff
@@ -96,7 +96,9 @@ <h2>
                                     <span class="text-muted">
                                         Game Version:
                                     </span>
-                                    {{ latest.gameversion.friendly_version }}
+                                    {% if latest %}
+                                        {{ latest.gameversion.friendly_version }}
+                                    {% endif %}
```

Comment:
This leaves empty lines as is:
```
                                <h2>
                                    <span class="text-muted">
                                        Game Version:
                                    </span>
                                    
                                        1.0
                                    
                                </h2>
```

```suggestion
                                    {% if latest -%}
                                        {{ latest.gameversion.friendly_version }}
                                    {%- endif %}
```